README.md

```diff
@@ -16,33 +16,44 @@ gem 'acts_as_scrubbable'
 
 ## Usage
 
-Simple add the configuration for your fields that map directly to your columns
+Add the configuration for your fields that map directly to your columns and a scrub_type
+for those columns.
 
+Default Scrub types include:
+- `scrub` - scrub the field's value based on it's name or mapping (see mapping case below)
+- `skip` - do not scrub the field's value
+- `wipe` - set the field's value to nil on scrub
+- `sterilize` - delete all records for this model on scrub
 
 ```ruby
-class User < ActiveRecord::Base
+class ScrubExample < ActiveRecord::Base
   ...
 
-  acts_as_scrubbable :first_name, :last_name
+  acts_as_scrubbable :scrub, :first_name # first_name will be random after `scrub!`
```

Comment:
i feel like "faker-generated" is clearer than "random". i had to look at the source code to figure that out.